src/main/DynamicConfig.ts

```diff
@@ -345,7 +345,7 @@ export class DynamicConfig implements IDynamicConfig {
             return this.buildDefaultForPlaceholder(placeholder)
 
         } else {
-            return resolver.get(placeholder.key).then(
+            return resolver.get(placeholder.key, placeholder.type).then(
```

Comment:
Does the `resolver.get()` call in `getValueFromResolver` need similar treatment?  Currently it doesn't have any handling for config value types.

Reply:
Good catch. That method is really for getting raw values from the remotes so it doesn't have access to the placeholder data. I added an optional param to pass the expected type through.